docs/usage/create.rst.inc

```diff
@@ -36,10 +36,10 @@ Exclusion options
         | read exclude patterns from EXCLUDEFILE, one per line
     ``--exclude-caches``
         | exclude directories that contain a CACHEDIR.TAG file (http://www.brynosaurus.com/cachedir/spec.html)
-    ``--exclude-if-present FILENAME``
-        | exclude directories that contain the specified file
-    ``--keep-tag-files``
-        | keep tag files of excluded caches/directories
+    ``--exclude-if-present PATH``
```

Comment:
while writing above comment, i noticed i was wrong suggesting the usage of "PATH" here, sorry.
it must not be a path (like /a/b/c), just a NAME.